Rakefile

```diff
@@ -4,3 +4,29 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+if ENV['IN_DOCKER']
+  def wait_for_db_container
+    loop do
+      return if connect_to_db
+      sleep 1
```

Comment:
I think you could add a counter here, and if you hit a time (5 minutes?) raise an exception.  I'd assume that makes the tests fail.

Reply:
Yeah travis kills us after 10 minutes. Up to you whether we should continue to just let that happen.

Reply:
I'm not a huge fan of requiring users to ctrl+c after some amount of time, I'd rather an error that says what the problem probably is (ie can't connect to postgres).

Reply:
+1, updated